slybot/slybot/plugins/scrapely_annotations/builder.py

```diff
@@ -308,12 +309,33 @@ def apply_selector_annotations(annotations, target_page):
             elems = accepted_elements - rejected_elements
         else:
             elems = [elem._root for elem in page.css(annotation['selector'])]
-        if elems:
-            tagids = [int(e.attrib.get('data-tagid', 1e9)) for e in elems]
-            tagid = min(tagids)
-            if tagid is not None:
-                annotation['tagid'] = tagid
-                converted_annotations.append(annotation)
+        if not elems:
+            continue
+
+        tagids = [int(e.attrib.get('data-tagid', 1e9)) for e in elems]
+        tagid = min(tagids)
+        if tagid is not None:
+            annotation['tagid'] = tagid
+            converted_annotations.append(annotation)
+        if (annotation.get('repeated') and
+                not annotation.get('item_container') and
+                len(annotation.get('annotations')) == 1):
```

Comment:
Can you add a local variable describing what this conditional represents? I'm not sure what this case corresponds to so the variable would reveal the intent here.